src/goto-instrument/accelerate/scratch_program.cpp

```diff
@@ -35,7 +35,7 @@ bool scratch_programt::check_sat(bool do_slice)
   output(ns, "scratch", std::cout);
 #endif
 
-  goto_symex_statet::propagationt::valuest constants;
```

Comment:
:question: This doesn't actually seem to be used anywhere, can it be removed?

Reply:
:+1: good catch